content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -0,0 +1,207 @@
+    1. Permitted x509 extensions - Non-critical extensions should be dropped.
+    1. Permitted key usages - must include []string{"client auth"}. Must not include key usages beyond []string{"digital signature", "key encipherment", "client auth"}
+    1. Expiration/cert lifetime - minimum of CSR signer or request. Sanity of the time is the concern of the signer.
+    1. CA bit allowed/disallowed - not allowed.
+
+1. `kubernetes.io/kube-apiserver-client-kubelet`: signs client certificates that will be honored as client-certs by the
+  kube-apiserver.
+  May be auto-approved by kube-controller-manager.
+    1. Trust distribution: signed certificates must be honored as client-certificates by the kube-apiserver.  The CA bundle
+       is not distributed by any other means.
+    1. Permitted subjects - organizations are exactly `[]string{"system:nodes"}`, common name starts with `"system:node:"`
+    1. Permitted x509 extensions - none
+    1. Permitted key usages - exactly `[]string{"key encipherment", "digital signature", "client auth"}`
+    1. Expiration/cert lifetime - minimum of CSR signer or request.  Sanity of the time is the concern of the signer.
+    1. CA bit allowed/disallowed - not allowed.
+
+1. `kubernetes.io/kubelet-serving`: signs serving certificates that are honored as a valid kubelet serving certificate
+  by the kube-apiserver, but has no other guarantees.
+  Never auto-approved by kube-controller-manager.
+    1. Trust distribution: signed certificates must be honored by the kube-apiserver as valid to terminate connections to a kubelet.
+       The CA bundle is not distributed by any other means.
+    1. Permitted subjects - organizations are exactly `[]string{"system:nodes"}`, common name starts with `"system:node:"`
+    1. Permitted x509 extensions - only DNS and IP SANs are allowed, at least one DNS or IP SAN must be present
+    1. Permitted key usages - exactly `[]string{"key encipherment", "digital signature", "server auth"}`
+    1. Expiration/cert lifetime - minimum of CSR signer or request.
+    1. CA bit allowed/disallowed - not allowed.
+
+1. `kubernetes.io/legacy-unknown`:  has no guarantees for trust at all. Some distributions may honor these as client
+  certs, but that behavior is not standard kubernetes behavior.
+  Never auto-approved by kube-controller-manager.
+    1. Trust distribution: None.  There is no standard trust or distribution for this signer in a kubernetes cluster.
+    1. Permitted subjects - any
+    1. Permitted x509 extensions - honors SAN extensions and discards other extensions.
+    1. Permitted key usages - any
+    1. Expiration/cert lifetime - minimum of CSR signer or request.  Sanity of the time is the concern of the signer.
+    1. CA bit allowed/disallowed - not allowed.
+
+Today, failures for all of these are only reported in kube-controller-manager-logs.
```

Comment:
[Avoid making statements or giving hints about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) (remove “Today, ”)